src/chromo/common.py

```diff
@@ -543,6 +537,79 @@ def __getnewargs__(self):
         # upon unpickling, create EventData object instead of MCEvent object
         return (EventData,)
 
+    @abstractmethod
+    def _add_init_beam_info(self):
+        pass
```

Comment:
Remove the abstractmethod decorator, so that this is the default implementation.